File: projects/mock/Demo_Mock/Hello_JS/api1/mock.js

function HandleRequest(req, res, state) {
    var statecount = state.get("counter");

    if (statecount) {
        statecount.count  = statecount.count + 1;
        state.save("counter", statecount);
    } else {
        statecount = {
            count: 1
        };
        state.save("counter", statecount);
    }
    
      var myres = {
        hello: 'hello world',
        counter: statecount.count,
        array : ["aa", "bb", "cc"],
    }

    res.json(myres);

}

module.exports = HandleRequest

File: projects/mock/Demo_Mock/Hello_GoLang/api0/mock.go
package api

import (
	"State"
	"fmt"
	"net/http"
)

type MyCounter struct {
	Count int `json:"count"`
}

func HandleRequest(w http.ResponseWriter, r *http.Request, pam map[string]string) error {

	// set or add headers
	w.Header().Set("aa", "123")
	w.Header().Add("bb", "456")

	//get/set Stat
	var counter = &MyCounter{}

	if err := State.Get("counter", &counter); err != nil {
		counter = &MyCounter{Count: 1}
	} else {
		counter.Count++
	}

	State.Save("counter", *counter)

	fmt.Fprintf(w, "Hello, "+pam["user"]+"\ncurrent counter:%d\n", counter.Count)

	return nil
}
